src/components/Posts/NewPost.style.js

```diff
@@ -0,0 +1,100 @@
+import styled from "styled-components";
+
+const NewPostCont = styled.form(
+  ({ openPost }) => `
+  max-width: 500px;
+  border: 1px solid steelblue;
+  padding: 30px;
+  border-radius: 5px;
+  flex-direction: column;
+  display: ${openPost ? "flex" : "none"};
+  box-shadow: 10px 10px 5px #aaaaaa;
+  border-radius: 5%; 
+  background-color:white;
+  margin-left:69rem
+`
+);
+
+const NewPostHeader = styled.h1`
+  font-size: 30px;
+  margin-left: 7rem;
+  color: steelblue;
+`;
+
+const InputName = styled.label`
+  display: block;
+  color: steelblue;
+  margin-left: 4px;
+`;
+
+const TitleInputContainer = styled.input`
+  width: 100%;
+  height: 40px;
+  margin: 5px;
+  padding: 3px;
+  font-size: 17px;
+`;
+
+const BodyContainer = styled.textarea`
+  width: 100%;
+  height: 120px;
+  margin: 5px;
+  padding: 3px;
+  font-size: 17px;
+  resize: none;
+`;
+
+const NewPostBtn = styled.button`
+display: inline-block;
+   background: steelblue;
+   color: #fff;
+   border: none;
+   padding: 10px 20px;
+   margin-left: 1.1rem;
+   border-radius: 5px;
+   cursor: pointer;
+   -webkit-text-decoration: none;
+   text-decoration: none;
+   font-size: 15px;
+   font-family: inherit;
+   right: 41rem;
+   top: 28.5rem;
+   
+}
```

Comment:
keep the right tabulation, please check other files too, run prettier to be sure everything is okay, you can use this comand ```prettier --write \"**/*.+(js|jsx|json|css|md)\"``` in your terminal for that and have a line break in the end of every file